### PR TITLE
Here we go again

### PR DESCRIPTION
**Pull request by skbolton**
I made a little mess of things so I wanted to back out that last pull and submit it better. Sorry I just had a strong beer and it made me a little sluggish lol

### CODE REVIEW

**Comment by sandman45** [comment]
Looks good but maybe we try catch config.silent so it doesn't break if anyone doesn't put in a silent option in config ?

**Comment by skbolton** [comment]
How about defaulting silent to false like in my last commit?

**Comment by sandman45** [comment]
ok, but doesn't trying to use config.silent throw an error if silent is not on the object?

**Comment by sandman45** [comment]
with the version I do a git release and upload it and roll the version on package.json

File: index.js [hunk 1]
```diff
@@ -59,6 +60,7 @@ var callbacks = {
 function Pool( _config, _callbacksConfig ) {
   if( _config ){
     config = _config;
+    silent = config.silent;
```
Comment (sandman45) [inline index.js]: maybe here we do something like this
```

if (config.silent) {
   silent = config.silent
}
```

Comment (skbolton) [reply index.js]: That or 
```
silent = config.silent || false;
```
Which do you like more?

Comment (sandman45) [reply index.js]: your suggestion seems cleaner... lets go that route

**Comment by sandman45** [comment]
looks good.. Ill merge and then update the release

**Pull request merged**